### PR TITLE
💪 Upcoming events - Improving loading time

### LINKED ISSUE #2578: ✨ Upcoming Events - Implement ISR
@wicksipedia 

https://www.ssw.com.au

**Description:**

Currently, we're displaying upcoming events on the page, fetching them on the client side. However, this approach introduces a delay message on loading and causes content shifting, potentially impacting the page load performance. I'm considering fetching these events from the server side instead to mitigate these issues

**Describe alternatives you've considered**

By fetching the upcoming events either during build time or implementing ISR with a cache of 1 hour, we can eliminate the need for users to see loading content. This improvement not only enhances the user experience but also boosts the page loading score.

- [x] Implement ISR for upcoming events to revalidate in 1 hour.

Image

**Figure: Upcoming events on the homepage**

### PR DESCRIPTION
**Pull request by amankumarrr**
Fixed #2578 

✅ The initial load time has decreased from 12-15 seconds to 1-2 seconds, and on subsequent loads, the content displays instantly.

Screenshot:

DD2D6CBF-C8A4-4DF1-91E8-6E900C3FDD42

**GIF: Loading upcoming events instantly**

### CODE REVIEW

File: pages/[...filename].tsx [hunk 1]
```diff
@@ -15,6 +15,7 @@ import { Container } from "../components/util/container";
 import { Section } from "../components/util/section";
 import { SEO } from "../components/util/seo";
 import { removeExtension } from "../services/client/utils.service";
+const ISR_TIME = 60 * 60; // 1 hour;
```
Comment (jeoffreyfischer) [inline pages/[...filename].tsx]: to remove the comment, you could rename ISR_TIME to ISR_TIME_SECONDS

**Comment by jeoffreyfischer** [comment]
✅ Tested on the slot - the Upcoming Events loaded instantaneously!

**Review by jeoffreyfischer — APPROVED** [review]
LGTM

File: pages/[...filename].tsx (hunk 1 above)
Comment (amankumarrr) [reply pages/[...filename].tsx]: Done - comments removed!

**Pull request merged**